docs/configuration.md

```diff
@@ -227,9 +227,9 @@ Defines some pre-configured world settings for the server.
 - COMPLETE_DARKNESS
 - DST_CAVE
 
-**WORLD_OVERRIDES**  
-Sets the overrides-configuration for world generation. Basically it's just the content for the
-`worldgenoverride.lua` file. As this value can be pretty large it's recommended to put the
+**LEVELDATA_OVERRIDES**  
```

Comment:
Changed it. Also, we still don't have a final release yet, so breaking changes for minor versions are fine for now, I believe.